tester-macos/bin/app.re

```diff
@@ -142,14 +131,16 @@ let () = {
     window#setContentView(root.view);
 
     window#windowDidResize(_ =>
-      Brisk.RunLoop.setWindowHeight(window#contentHeight)
+      Brisk.UI.setWindowHeight(window#contentHeight)
     );
 
-    Brisk.RunLoop.renderAndMount(
+    Brisk.UI.renderAndMount(
       ~height=window#contentHeight,
       root,
       Brisk.element(<Component />),
     );
+
+    Brisk.RunLoop.spawn();
```

Comment:
Can we call it something other than `spawn`?

Reply:
I mean, spawn is a perfect name:
https://en.wikipedia.org/wiki/Spawn_(computing)